docs/use-cases/image-data.md

```diff
@@ -69,18 +70,37 @@ For example, set it to `0.05` to capture an image every 20 seconds.
 
 Set the MIME type to your desired image format, for example `image/jpeg`.
 
+**Click save to start capturing data.**
```

Comment:
Why is this bold?

Reply:
It's quite far down in the step, so worried people will miss it/forget